ibl_pipeline/behavior.py

```diff
@@ -128,16 +128,32 @@ class WheelMoveType(dj.Lookup):
     contents = [['CW'], ['CCW'], ['Flinch'], ['Other']]
 
 
+@schema
+class CompleteWheelMoveSession(dj.Computed):
+    definition = """
+    # sessions that are complete with wheel related information and thus may be ingested
+    -> acquisition.Session
+    ---
+    wheelmove_session_complete: bool              # whether the session is complete
+    """
+
+    required_datasets = ["_ibl_wheelMoves.intervals.npy", "_ibl_wheelMoves.type.csv"]
+
+    def make(self, key):
+        datasets = (data.FileRecord & key & 'repo_name LIKE "flatiron_%"' & {'exists': 1}).fetch('dataset_name')
+        key['wheelmove_session_complete'] = bool(np.all([req_ds in datasets for req_ds in self.required_datasets]))
```

Comment:
The following is more efficient because it stops as soon as the first `False` is detected.
```python
all(req_ds in datasets for req_ds in self.required_datasets)
```